### PR TITLE
Bugfix - review applicants french copy

### LINKED ISSUE #2882: Bug - Use defineMessage function with ReviewApplication text
# Description
Some of the text in the ReviewApplications components has not been declared via the `defineMessages` react-intl function, so it has never shown up during an `npm run translations` run, which means it has never been translated.

For example, lines 90-154 in ReviewCategory.tsx.

Translations will be required once the text is properly integrated into the setup.

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #2882

### CODE REVIEW

File: resources/assets/js/localizations/fr.json [hunk 1]
```diff
@@ -18,26 +18,12 @@
   "apl.expandAllSkills": "Élargir les compétences",
```
Comment (Khristophor) [inline resources/assets/js/localizations/fr.json]: I don't think this file is actually being used by anything anymore, but I'm not confident enough to say remove it

Comment (yonikid15) [reply resources/assets/js/localizations/fr.json]: Hmm i'm not sure either, but I can find out.

Comment (yonikid15) [reply resources/assets/js/localizations/fr.json]: Hey, so I was able to remove most of the message data from the `localizations` folder. The remaining messages are needed in the `alert.js` file, which is used for deleting draft job posters in the Manager job index page. There weren't that many messages so I just moved them into the file itself and removed the folder.

File: resources/assets/js/alerts.js [hunk 2]
```diff
@@ -86,16 +98,9 @@ function handleClick(e) {
 
 const indexWrapper = document.querySelector(".manager-poster-index");
 // Double check we're on the manager index page, grab all the
-// 'Send for Review' and 'Delete' buttons, and register the click event handler
+// 'Delete' buttons, and register the click event handler
 // defined above.
 if (typeof indexWrapper !== "undefined" && indexWrapper != null) {
-  const reviewButtons = Array.from(
```
[the remark this answers is not in the document]
Comment (yonikid15) [reply resources/assets/js/alerts.js]: We don't use this anymore actually, the send for review button has been moved to the Review step in the JPB.

[the remark this answers is not in the document]
Comment (Khristophor) [reply resources/assets/js/alerts.js]: Screen Shot 2020-02-13 at 12 04 48 PM
This is on the manager index view of the current branch

[the remark this answers is not in the document]
Comment (yonikid15) [reply resources/assets/js/alerts.js]: So the Send for Review button is now a link, that takes you the Review step in the JPB. It used to be a button that opened the sweet alert modal. That's the content that I removed here since were not using it anymore.

File: resources/assets/js/components/ApplicationReview/ApplicationReview.tsx [hunk 3]
```diff
@@ -348,19 +360,15 @@ class ApplicationReview extends React.Component<
...
+              {intl.formatMessage(messages.viewApplicationText)}
             </a>
             <a
               title={intl.formatMessage(messages.viewProfileTitle)}
               href={applicantUrl}
             >
               <i className="fas fa-user" />
               <FormattedMessage
-                id="apl.viewProfile"
+                id="application.review.viewProfile"
```
Comment (tristan-orourke) [inline resources/assets/js/components/ApplicationReview/ApplicationReview.tsx]: Why did you change other instances of <FormattedMessage> to `intl.formatMessage()` but keep this one? Its fine, just inconsistent.

Comment (yonikid15) [reply resources/assets/js/components/ApplicationReview/ApplicationReview.tsx]: Oops, missed this one.
